### ISSUE TITLE
Importing divviup-ts from Node

### ISSUE THREAD

**Issue by divergentdave** [divviup/divviup-ts#89]
I ran into some issues when trying to import the divviup-ts library from plain Node, and I'm wondering if there's some combination of TypeScript settings we could use to improve this. It appears we are using TypeScript's classic module resolution strategy, and internal module dependencies use "non-relative module imports". I got things to work by changing all internal dependencies to use relative imports, (a very invasive change, which we may not want to do) and importing the result as follows.

```json
{
  "dependencies": {
    "divviup-ts": "file:../../Code/ppm/divviup-ts/dist/src"
  }
}
```

```js
const DAPClient = require("divviup-ts/dap").default;
```

**Comment by divergentdave** [comment]
These links provide a good overview of the situation: https://stackoverflow.com/a/57539446, https://github.com/Microsoft/TypeScript/issues/15479#issuecomment-300240856, and https://github.com/Microsoft/TypeScript/issues/15479#issuecomment-468122916. Compiler options will only affect the compiler's internal resolution of imports, but won't affect code generation.

We are presented with a choice between exclusively using relative module import paths or adding another build step after `tsc` to transforms imports or patches resolution. The TypeScript core developer in the above issue thread appears to advocate for using relative import paths when Node.js is a target. On the other hand, non-relative module imports are plainly much more readable and easier to work with. Our modules nest three or four layers deep, and we have a lot of import statements pointing up multiple directories.

I think this would be best resolved via Parcel, following #34. It appears that the ["alias" field](https://en.parceljs.org/module_resolution.html#aliases) can address this.

**Comment by divergentdave** [comment]
#34 has solved most of this, the only remaining hiccup is that the `DAPClient` constructor is exported under the name `default`. Here's my updated code sample.

```json
{
  "dependencies": {
    "@divviup/dap": "file:../../Code/ppm/divviup-ts/packages/dap"
  }
}
```

```js
const DAPClient = require("@divviup/dap").default;

async function main() {
  const client = new DAPClient({
    type: "sum",
    bits: 2,
    taskId: ...,
    leader: ...,
    helper: ...,
    minBatchDurationSeconds: 300,
  });

  console.log("Start time is", Math.floor(new Date().getTime() / 1000 / 300) * 300);
  for (var i = 0; i < 10; i++) {
    await client.sendMeasurement(1);
  }
  console.log("Done uploading");
  console.log("End time is", Math.ceil(new Date().getTime() / 1000 / 300) * 300);
}

main().catch(console.error);
```
